integrations/kube-agent-updater/go.mod

```diff
@@ -5,18 +5,18 @@ go 1.19
 require (
 	github.com/Masterminds/sprig/v3 v3.2.3
 	github.com/docker/distribution v2.8.2+incompatible
-	github.com/google/go-containerregistry v0.14.0
+	github.com/google/go-containerregistry v0.15.2
 	github.com/gravitational/trace v1.2.1
-	github.com/jonboulle/clockwork v0.3.0
+	github.com/jonboulle/clockwork v0.4.0
```

Comment:
The 0.4.0 update required a lot of manual fixes in Teleport. Are we sure no changes are necessary here?

Reply:
I left this update in because `make test-kube-agent-updater` seems to be passing, I will watch for other unit test failures.